spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/AboutControllerTests.java

```diff
@@ -343,7 +345,7 @@ public void testChecksumNoDefaults() throws Exception {
 	}
 
 	@ExtendWith(SpringExtension.class)
-	@SpringBootTest(classes = TestDependencies.class)
+	@SpringBootTest(classes = {TestDependencies.class, ProjectInfoAutoConfiguration.class, InfoContributorAutoConfiguration.class},properties = {"management.info.git.enabled=true"})
```

Comment:
**[TinyNit]** spacing before/after the `{` and `}` and put the `properties` on its own line.